src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -601,3 +601,17 @@ void MainPage::AnnounceCategoryName()
     NarratorAnnouncement^ announcement = CalculatorAnnouncement::GetCategoryNameChangedAnnouncement(categoryName);
     NarratorNotifier->Announce(announcement);
 }
+
+void MainPage::OnNavItemInvoked(_In_ Microsoft::UI::Xaml::Controls::NavigationView^ sender, _In_ Microsoft::UI::Xaml::Controls::NavigationViewItemInvokedEventArgs^ e)
+{
+    auto item = dynamic_cast<MUXC::NavigationViewItem^>(e->InvokedItemContainer);
```

Comment:
I think it's safe to simplify this code and always call `NavView->IsPaneOpen = false;` without checking the value of Mode.

What's your opinion?

Reply:
I agree with this.

Reply:
Great! The less we use `DataContext` in the code, the better!